dev/validate_bom__config.py

```diff
@@ -961,7 +961,9 @@ def add_config(self, options, script):
     account_params = [options.k8s_account_name]
     account_params.extend([
         '--docker-registries', options.k8s_account_docker_account,
-        '--kubeconfig-file', os.path.basename(options.k8s_account_credentials)
+        '--kubeconfig-file', os.path.basename(options.k8s_account_credentials),
+        '--provider-version',
+        'v1'
```

Comment:
for consistency, move `'v1'` up a line?

Reply:
(But also, who cares, this is just getting deleted soon anyway, I DUNNO)

Reply:
I care! Moved it up!